Public/Src/IDE/VsCode/client/src/xlg-status.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+import * as vscode from "vscode";
+import { DocumentColorParams } from "vscode-languageserver-protocol/lib/protocol.colorProvider.proposed";
+
+export const BxlStatusCsvFileName = "BuildXL.status.csv";
+export const CmdRenderStatus = "XLG.Render.Status";
+export const DefaultColumnsOfInterest = [ "*" ];
+export const DefaultColumnsToRender = [ "Cpu Percent", "Mem Percent", "Process Running" ];
+
+export function activate(context: vscode.ExtensionContext) {
+    context.subscriptions.push(vscode.commands.registerCommand(CmdRenderStatus, () => {
+        return renderActiveBxlStatusCsv();
+    }));
+}
+
+function range(start: number, count: number): number[] {
+    return Array(count).fill(0).map((elem, idx) => start + idx);
+}
+
+function splitColumns(line: string): string[] {
+    return line.split(",").map(s => s.trim());
+}
+
+function convertBxlTimeToSeconds(bxlTimeValue: string) {
+    if (bxlTimeValue.startsWith("["))
+        bxlTimeValue = bxlTimeValue.substring(1);
+    if (bxlTimeValue.endsWith("]"))

```

Comment:
It seems like in other places we have been surrounding if/else statements with brackets even if there is only 1 line in the statement (ie line 88 or 121) . Should we standardize that everywhere or does TS have some different formatting rules?

Reply:
I don't think we have a strict standard for writing .ts files, so I'll leave it as is.